Yoast/wordpress-seo
# Remove WordProof integration

## Context


* Removes WordProof integration completely.

## Summary


This PR can be summarized in the following changelog entry:

* Removes WordProof integration completely.

## Relevant technical choices:

*

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

*


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* On the SEO settings -> General -> Integrations, verify that the WordProof integration toggle has disappeared.
* On the privacy page, in the Advanced Settings tab, verify that the WordProof toggle has disappeared.
* On the privacy page, check the Network tab and verify that no `wordproof` related assets are being loaded.
* Regression/smoke test SEO general settings and SEO page/post settings.

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


Fixes #
